images/php/7.2/Dockerfile

```diff
@@ -44,6 +44,20 @@ RUN docker-php-ext-install \
   xsl \
   zip
 
+RUN cd /tmp \
```

Comment:
Can you keep just this single `RUN` command and remove all of the other updates? We don't want to add backports to the repo to add libsodium, it's already added in libsodium-dev above.

Reply:
Whoops.  My bad.  Thanks for catching that.  I have made the requested changes.  Thanks!